src/App.css

```diff
@@ -76,3 +76,27 @@ label {
   padding: 2px;
   margin: 10px;
 }
+
+.orange-button {
```

Comment:
Good call moving this code to Button.css! Can this be removed from here now?

Reply:
Yes, thank you for catching that! I forgot to delete. Done.